runtime/vm/classallocation.c

```diff
@@ -93,7 +97,10 @@ internalAllocateClassLoader(J9JavaVM *javaVM, j9object_t classLoaderObject)
 	J9VMThread *vmThread = currentVMThread(javaVM);
 	UDATA i = 0;
 	BOOLEAN allocRetried = FALSE;
-	J9Class *knownClasses[CLASS_PROPAGATION_TABLE_SIZE];
+	J9Class *knownClasses[CLASS_PROPAGATION_TABLE_SIZE] = {0};
+#if JAVA_SPEC_VERSION >= 9
```

Comment:
To date, all checks for java version are done using runtime checks instead of compile time checks. Please use a runtime check instead.

Reply:
added runtime checks.